src/ert/scheduler/job.py

```diff
@@ -187,6 +187,13 @@ async def _handle_failure(self) -> None:
             f"\n\t{self._callback_status_msg}"
         )
 
+        if msg := self.driver._job_error_message_by_iens.get(self.iens, ""):
+            error_msg += f"\n\tDriver reported: {msg}"
```

Comment:
I am not sure I like the tab-character here..

Reply:
I don't think we need the empty string in `...get(self.iens, "")` either.

Reply:
`if empty string` yields False

Reply:
I can remove the tab though :+1: